swayidle/swayidle.1.scd

```diff
@@ -0,0 +1,59 @@
+swayidle (1)
+
+# NAME
+
+swayidle - Idle manager for Wayland
+
+# SYNOPSIS
+
+*swayidle* [options] [events...]
+
+# OPTIONS
+
+*-h*
+	Show help message and quit.
+
+*-d*
+	Enable debug output.
+
+# DESCRIPTION
+
+swayidle listens for idle activity on your Wayland compositor and executes tasks
+on various idle-related events. You can specify any number of events at the
+command line.
+
+#EVENTS
+
+*timeout* <timeout> <timeout command> [resume <resume command>]::
+	Execute <timeout command> if there is no activity for <timeout> seconds.
+	+
+	If you specify "resume <resume command>", <resume command> will be run when
+	there is activity again.
+
+*before-sleep* <command>::
+	If built with systemd support, executes <command> before systemd puts the
+	computer to sleep.
+
+All commands are executed in a shell.
+
+# EXAMPLE
+
+ swayidle \
+     timeout 300 'swaylock -c 000000' \
+     timeout 600 'swaymsg "output * dpms off"' \
+		resume 'swaymsg "output * dpms on"' \
+     before-sleep 'swaylock -c 000000'
```

Comment:
This should be in a literal block, use \`\`\` (see man scdoc)

Reply:
Done